banjo/printer.cpp

```diff
@@ -481,6 +482,17 @@ Printer::postfix_type(Array_type const& t)
 }
 
 
+// FIXME: This is most likely very wrong
+void
+Printer::postfix_type(Tuple_type const& t)
+{
+  postfix_type(t.type());
+  token(lbrace_tok);
+  expression(t.extent());
```

Comment:
You need to parse a comma delimited list of types. Look at expression-list in parse-expr.cpp.